packages/itwinui-css/src/table/table.scss

```diff
@@ -254,14 +254,13 @@
       }
     }
 
+    &.iui-loading,
     .iui-table-cell[aria-disabled='true'],
     &.iui-table-expanded-content[aria-disabled='true'] {
-      font-style: italic;
-      cursor: not-allowed;
       color: var(--iui-color-foreground-4);
 
-      &.iui-slot:hover > .iui-table-more-options {
-        opacity: 0;
+      .iui-table-more-options {
+        display: none;
```

Comment:
This was made `opacity: 0` to keep it tabbable through keyboard

Reply:
When I changed it to `opacity: 0`, it allows tabbing through the loading's row's more options. Not sure if that is intended, because we don't allow tabbing through the textbox with "alpha.dgn"

[vokoscreenNG-2022-09-28_10-47-59.webm](https://user-images.githubusercontent.com/45748283/192811544-f3fd3a19-cde6-4527-af18-0545347797a5.webm)

Reply:
> When I changed it to `opacity: 0`, it allows tabbing through the loading's row's more options.

Yes, that is my comment. We want to keep the button accessible to keyboard users.

> we don't allow tabbing through the textbox with "alpha.dgn"

That is a separate issue that we should fix separately.

Reply:
Reverted the `display: none` to `opacity: 0` for table rows.  Now the more options button can be accessed in both the disabled and loading states.  Previously we were disabling the tile's more options button in disabled / loading states, the tile has been updated to align with table rows.